src/storage/storage.cpp

```diff
@@ -402,15 +313,15 @@ void Storage::PopulateData(const DataLayout &layout, char *memory_ptr)
     {
         const auto name_blocks_ptr =
             layout.GetBlockPtr<extractor::NameTableView::IndexedData::BlockReference, true>(
-                memory_ptr, DataLayout::NAME_BLOCKS);
+                memory_ptr, "/common/names/blocks");
```

Comment:
It is good to move initialization into a free function as
```
return util::vector_view<BlockT> (layout.GetBlockPtr<BlockT, true>(memory_ptr, name), layout.GetBlockEntries(name));
```
to avoid duplication of type-name pairs.

Reply:
Oh great idea. :+1: